### PR TITLE
Upgrade python setuptools

### PR DESCRIPTION
**Pull request by ShooterIT**

### CODE REVIEW

File: .github/workflows/kvrocks.yaml [hunk 1]
```diff
@@ -19,10 +19,15 @@ jobs:
...
         uses: actions/checkout@v2
 
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+            python-version: '3.x'
+
       - name: Install Dependencies
         run: |
+          sudo python -m pip install --upgrade pip
```
Comment (ShooterIT) [inline .github/workflows/kvrocks.yaml:29]:
```suggestion
          sudo -H python -m pip install --upgrade pip
```

File: .github/workflows/kvrocks.yaml [hunk 2]
```diff
@@ -19,10 +19,15 @@ jobs:
...
 
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+            python-version: '3.x'
+
       - name: Install Dependencies
         run: |
+          sudo -H python -m pip install --upgrade pip
           sudo apt-get install -y cppcheck
```
Comment (ShooterIT) [inline .github/workflows/kvrocks.yaml:30]:
```suggestion
          sudo apt-get update
          sudo apt-get install -y cppcheck
```

File: .github/workflows/kvrocks.yaml [hunk 3]
```diff
@@ -19,10 +19,16 @@ jobs:
...
         uses: actions/checkout@v2
 
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+            python-version: '3.x'
+
       - name: Install Dependencies
         run: |
+          sudo -H python -m pip install --upgrade pip
```
Comment (ShooterIT) [inline .github/workflows/kvrocks.yaml:29]:
```suggestion
          sudo pip install --upgrade pip
```

File: .github/workflows/kvrocks.yaml [hunk 4]
```diff
@@ -19,10 +19,16 @@ jobs:
...
 
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+            python-version: '3.x'
+
       - name: Install Dependencies
         run: |
-          sudo apt-get install -y cppcheck
           sudo pip install --upgrade pip
```
Comment (ShooterIT) [inline .github/workflows/kvrocks.yaml:29]:
```suggestion
          sudo pip install --upgrade pip
          sudo pip install --upgrade setuptools
```

File: .github/workflows/kvrocks.yaml [hunk 5]
```diff
@@ -19,10 +19,17 @@ jobs:
...
+          sudo pip install --upgrade setuptools
+          sudo apt-get update
```
Comment (ShooterIT) [inline .github/workflows/kvrocks.yaml:31]:
```suggestion
```

File: .github/workflows/kvrocks.yaml [hunk 6]
```diff
@@ -19,10 +19,17 @@ jobs:
       - name: Checkout Code Base
         uses: actions/checkout@v2
 
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+            python-version: '3.x'
+
```
Comment (ShooterIT) [inline .github/workflows/kvrocks.yaml:26]:
```suggestion
```

**Review by ShooterIT — COMMENTED** [review]
remove python3

File: .github/workflows/kvrocks.yaml [hunk 7]
```diff
@@ -21,8 +21,9 @@ jobs:
 
       - name: Install Dependencies
         run: |
-          sudo apt-get install -y cppcheck
           sudo pip install --upgrade pip
+          sudo pip install --upgrade setuptools
+          sudo apt-get install -y cppcheck
```
Comment (ShooterIT) [inline .github/workflows/kvrocks.yaml:26]:
```suggestion
          sudo apt-get update
          sudo apt-get install -y cppcheck
```

**Review by git-hulk — APPROVED** [review]
LGTM

**Pull request merged**